tensorboard/components_polymer3/vz_sorting/test/BUILD

```diff
@@ -11,6 +11,7 @@ tf_web_test(
     name = "test",
     src = "/vz-sorting/test/tests.html",
     web_library = ":test_web_library",
+    tags = ["manual"]
```

Comment:
buildifier? (trailing comma is missing.)

Reply:
Done, trailing comma and order was off